src/aWOT.cpp

```diff
@@ -257,405 +256,324 @@ void Response::m_printStatus(int code) {
   switch (code) {
 #ifndef LOW_SRAM_MCU
     case 100: {
-      P(Continue) = "Continue";
-      printP(Continue);
       break;
     }
     case 101: {
-      P(SwitchingProtocols) = "Switching Protocols";
-      printP(SwitchingProtocols);
```

Comment:
Was there a reason to remove this? The tests are now failing because of this. I need to fix the ci integration to prevent this from happening.